style/ferry-routes.mss

```diff
@@ -4,8 +4,10 @@
 #ferry-routes {
   [zoom >= 8] {
     /* background prevents problems with overlapping ferry-routes, see #457 */
-    background/line-color: @water-color;
+    background/line-color: @river-color;
     background/line-width: 1; /* Needs to be a bit wider than the route itself because of antialiasing */
+    background/line-comp-op: lighten;
+    background/line-dasharray: 4,4;
```

Comment:
The idea of a background line is that it covers the dashes that are in an different position from another ferry line. Using dashes for the background line defeats the purpose, it should just be solid.